dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/sme/SmeFramework.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.frameworktoolbox.frameworks.sfdr
+
+import org.dataland.frameworktoolbox.frameworks.InDevelopmentPavedRoadFramework
+import org.springframework.stereotype.Component
+import java.io.File
+
+/**
+ * Definition of the SME framework
+ */
+@Component
+class SmeFramework : InDevelopmentPavedRoadFramework(
+    identifier = "sme",
+    label = "SME",
+    explanation = "Small and medium-sized enterprises questionnaire",
+    File("./dataland-framework-toolbox/inputs/sme/sme.xlsx"),
+    order = 4,
```

Comment:
Order is the same as for p2p now.

Reply:
Ja stimmt, macht keinen Sinn.

Ich hab jetzt die orders von allen frameworks nochmal überarbeitet, und jedes hat jetzt eine eigene order-number.

Ich habe die deutschsprachigen (wie einst von Julia gewünscht) ans Ende der Liste gepackt, und sme einfach direkt davor.
Finde das macht Sinn.

Reply:
Sehr gut